src/tools/id-mapping/components/results/__mocks__/tooManyIDsForMappingDetails.ts

```diff
@@ -0,0 +1,12 @@
+export default {
+  from: 'UniProtKB_AC-ID',
+  to: 'GI_number',
+  ids: 'A0A024B7W1,A0A024RBG1,and many more',
+  errors: [
+    {
+      code: 40,
+      message:
+        'Id Mapping API is not supported for mapping results with "mapped to" IDs more than 500000',
```

Comment:
```suggestion
        'ID Mapping API is not supported for mapping results with "mapped to" IDs more than 500 000',
```
I would maybe suggest a phrasing that might make more sense to users like "ID Mapping API is not supported when there are more than 500 000 results". Additionally we could suggest to reduce the input size

Reply:
That makes me think, we could also provide an automatic batching of queries in the website. We have an input limit, but what we could provide is a batch option in the form, automatically going higher if the user goes over the limit, and from this error message we could generate a link to the input form pre-filled with a higher number in order to split the search even more

Reply:
That's a nice idea. I'll create a few jiras:

1. Ask backend to update their returned error message as you've indicated
2. Create a batch id mapping submission workflow in the website

Reply:
1. Ah, I didn't realise that this message was coming directly from the backend message. Then that shouldn't block this PR I guess
2. Yes, that could be for both. Straight from the input length limit, we could submit multiple jobs, and in the case of an issue of too many results, then do the iterative approach of suggesting to resubmit through a link with a higher number of jobs. If additionally to the "too many results" we could actually know how many results there are we could actually know how many jobs to submit